common-authentication/src/main/java/org/alfresco/hxi_connector/common/adapters/messaging/repository/ApplicationInfoProvider.java

```diff
@@ -52,8 +52,10 @@ public String getUserAgentData()
     {
         if (StringUtils.isBlank(applicationInfo))
         {
+            log.debug("Calculating user agent data.");
```

Comment:
```suggestion
            log.atDebug().log("Calculating user agent data.");
```

Reply:
I went with `log.debug` approach where `log.atDebug()` is not necessary.